domains/dave.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "davad00",
+           "email": "",
+           "discord": "1201678865093894234"
+        },
+    
+        "record": {
+            "CNAME": "dave-zrnu.onrender.com"
```

Comment:
"Not Found" are you able to provide a preview of the page?

Reply:
i am currently working on it..
i belive ill deploy it in a few hours..

Reply:
i deployed it :_)

heres the new cname:
dave-uuvs.onrender.com

Reply:
i deployed it :)

heres the new cname:
dave-lezt.onrender.com
❤️

Reply:
i deployed it :)

heres the new cname:
dave-lezt.onrender.com
❤️